OracleTuxedo/core/samples/shm/buildContainerImage.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+docker build -t oracle/tuxedoshm .
```

Comment:
While you are right in the sense that all it does is "docker build" but the extra steps listed at the end in the form of "echo" statements could be useful for the client to figure out the next steps instead of again going to some README or some html page to figure it out.

Reply:
They had to find the README to create the image in the first place, and would most likely be working from it to create the same. Likewise, if someone knows how to run `docker build` then they can know how to run `docker run` too. 

If you want to keep the script, please make it Oracle Linux 8 compatible by making it Podman-aware, i.e. it should test to see if Podman is installed and use that instead of Docker.

Reply:
Currently our image as well as the Oracle Server JRE image which we are using is based on the Oracle Linux 7.